packages/core/src/codecatalyst/commands.ts

```diff
@@ -27,6 +27,7 @@ import { Auth } from '../auth/auth'
 import { SsoConnection } from '../auth/connection'
 import { isInDevEnv, isRemoteWorkspace } from '../shared/vscode/env'
 import { getShowManageConnections } from '../login/command'
+import { commandPalette } from '../codewhisperer/commands/types'
```

Comment:
this "source" is pretty general so should probably live in somewhere higher up, e.g. constants.ts